### DISCUSSION TITLE
finish the authorization at Home Connect?

### DISCUSSION THREAD

**Discussion by totti1171** [alexkn/node-red-contrib-homeconnect#31] [Q&A] [answered]
How I can "finish the authorization at Home Connect and then save and deploy your changes."?

**Comment by alexkn** [comment, marked as answer]
Home Connect authorization is opened in a new browser tab.
So you can finish the authorization there.
Then go back to your Node-Red tab and save the node properties and deploy.

I know this is a bit inconvenient, but I don't see a better way to do the authorization code grant flow.

**Reply by totti1171** [reply]
sorry - there is now new browser tab (in safari) - but in Chrome (security settings) ;-)

thx!
